lib/tasks/conditions/process_conditions.rake

```diff
@@ -47,9 +47,13 @@ namespace :shf do
         log.error('Retrying the previous condition...')
         retry
 
+      # .... Otherwise, an exception was raised during condition processing -
+      #      log the error and continue with condition processing.
+      #      (If Slack connection is up, we'll have already notified via that).
       rescue StandardError => e
+        log.error("Class: #{class_name}")
         log.error("Exception: #{e}:  #{e.inspect}")
-        raise
+        next
```

Comment:
I didn't know about using `next` and `rescue` with loops.  Now I do! 
 (And now I know that it was introduced in ruby 2.5)
[I bet there are other places that we can use this pattern.]